### DISCUSSION TITLE
스터디 2기 참가 수요 파악 (재수, 코치, 리더)

### DISCUSSION THREAD

**Discussion by DaleSeo** [DaleStudy/leetcode-study#180] [공지사항]
지난 회고 때 1기 스터디 멤버분들이 어떻게 2기 스터디에도 기여하실 수 있을지 논의해보았습니다. 크게 다음 3가지 형태가 될 것 같아요.

## 재수
- 다시 신입의 마음으로 리트코드 75문제를 한 번 더 풀고 싶은 분
- 풀었던 문제가 실제 코딩 테스트에 나왔을 때 스스로 해결하실 자신이 아직 없으신 분
- 1기 활동을 돌아보았을 때, 매주 마감에 쫓겨 문제에 대해서 충분히 고찰하시지 못하신 분들께 추천

## 코치 (선배)
- 동일한 10주의 과정을 먼저 겪어 본 선배의 입장에서 코드 리뷰 및 질의 응답에 적극적으로 참여하시고 싶은 분
- 다양한 답안 코드를 접하면서 문제에 접근하는 새로운 관점과 통찰을 얻고 싶은신 분
- 1기 활동을 나름 충실하게 하셨고 스터디의 좋은 경험을 다른 분들께 나누시고 싶은 분들께 추천

## 리더
- 스터디를 주도적으로 이끄고 싶으신 분 (1기의 @SamTheKorean와 @leokim0922 역할)
- 코드 저장소와 커뮤니티 관리를 하실 수 있도록 깃허브와 디스코드에서 maintainer 권한을 부여
- 그 밖에 각종 특해 ㅋㅋ

어떤 형태로든 2기 스터디를 도와주실 의향이 있으신 분들은 아래 댓글로 달아주시면 다음 기수 계획을 세우는데 도움이 될 것 같습니다! 🙏

**Comment by YeonguChoe** [comment]
리더 하고 싶습니다. 언제 시작하나요?

**Reply by DaleSeo** [reply]
@YeonguChoe 님, 본 공지는 1기 분들 대상으로 올린건데 착오가 좀 있으신 것 같네요. 1기가 차주에 종료될 예정이라서 2기 모집(대기자 우선 선발)을 차주부터 시작할 예정입니다. 2기 일정 확정되면 별도로 공지하도록 하겠습니다!

**Reply by YeonguChoe** [reply]
아하. 제가 불쑥 나타났군요. 죄송합니다.
2기 참가하고 싶습니다. 2기 공지 보면 댓글 달겠습니다.

**Comment by SamTheKorean** [comment]
선배 혹은 리더로 참여하고 싶습니다!

**Comment by dev-jonghoonpark** [comment]
저도 선배로 지원 해보겠습니다 🥰

**Comment by bky373** [comment]
저는 1기 활동하며 충분히 생각해보지 못하고 놓친 문제들이 있어 재수 희망합니다!

**Comment by nhistory** [comment]
선배로 참여희망합니다!

**Comment by leokim0922** [comment]
선배로 참여하겠습니다 😄

**Comment by sounmind** [comment]
선배로 참여하고 싶습니다~

**Comment by yolophg** [comment]
선배로 참여 희망합니다!

**Comment by sbk1014** [comment]
저는 최근 시작된 구직 활동 때문에 2기 참여는 어려울 것 같습니다!

**Reply by DaleSeo** [reply]
괜찮습니다 @bhyun-kim 님, 이렇게 친절하게 댓글 남겨주셔서 감사합니다. 구직 준비 화이팅 하세요! 💪
